Release tag builds fail with an error

```
Run curl -fsSL https://get.docksal.io | bash
  curl -fsSL https://get.docksal.io | bash
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    REPO: docksal/docksal
    DOCKSAL_SSH_AGENT_USE_HOST: 0
    DOCKSAL_DNS_DISABLED: 1
    DOCKSAL_DNS_DOMAIN: docksal.site
    DOCKSAL_VERSION: refs/tags/v1.16.1
Enabling Sandbox Server installation mode...
curl: (22) The requested URL returned error: 404 
Error: Process completed with exit code 22.
```

https://github.com/docksal/docksal/actions/runs/1283411000

--- a/bin/fin
+++ b/bin/fin
@@ -2,6 +2,11 @@
 
 FIN_VERSION=1.108.0
 
+# Allow to override Docksal version during install.
+# Install: curl -L https://get.docksal.io | DOCKSAL_VERSION=develop bash
+# Update: DOCKSAL_VERSION=develop fin update
+DOCKSAL_VERSION="${DOCKSAL_VERSION:-v1.16.2}"
+
 # Predefined terminal colors
 # Format: '\033[<formatting>;<text-color>;<background-color>m'
 # See https://misc.flogisoft.com/bash/tip_colors_and_formatting
@@ -183,11 +188,6 @@ ver_to_int ()
 }
 
 #---------------------------- Global Constants --------------------------------
-
-# Allow to override Docksal version during install.
-# Install: curl -L https://get.docksal.io | DOCKSAL_VERSION=develop bash
-# Update: DOCKSAL_VERSION=develop fin update
-DOCKSAL_VERSION="${DOCKSAL_VERSION:-master}"
 
 # Dependency versions
 REQUIREMENTS_DOCKER='20.10.12'
@@ -2459,7 +2459,8 @@ version ()
 	if [[ $1 == '--short' ]]; then
 		echo "$FIN_VERSION"
 	else
-		echo "fin version: $FIN_VERSION"
+		echo "Docksal version: $DOCKSAL_VERSION"
+		echo "fin version:     $FIN_VERSION"
 	fi
 
 	# Ping stats server
@@ -6070,7 +6071,7 @@ sysinfo ()
 	eval $(parse_params "$@")
 
 	echo
-	echo "███  FIN" &&
+	echo "███  DOCKSAL" &&
 	version
 
 	echo

